microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/advanced/misc/StaticFilesTest.java

```diff
@@ -192,10 +191,12 @@ public static WebArchive bigStaticFileLocalServicesRouterDeployment()
         try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
             configuredHTTPPort = OpenApiServerConfiguration.getHTTPPort(client);
         }
-        String props = String.format(
-                "mp.openapi.extensions.path=/big/openapi\nservices.provider.host=%s\nservices.provider.port=%d",
+        final Integer maximumFileSize = 64 * 1024 * 1024;
```

Comment:
Can you please elaborate this a little bit?

My understanding is that MP has small default maximum size and maximumStaticFileSize param increase the default value, right? If yes, what is the default size and what is the size of file used in this test?

Reply:
Hi @marekkopecky - yes, you've got it.
SnakeYaml has set a £ MB limit to avoid issues (a CVE was filed because of DoS when loading very big static files).
So there is this test which is building and loading a big one (should be several MBs, I don't recall now but bigger that 3 MB), which started to fail.
By leveraging the new `maximumStaticFileSize` property, we are increasing the limit so that the test is passing again.

Reply:
Thanks!